docs/guide/Alloy_Framework/Alloy_How-tos/Alloy_Reference_Guides/Alloy_Command-Line_Interface_Reference.md

```diff
@@ -9,7 +9,7 @@ The Alloy Command-Line Interface (CLI) is a command-line tool for managing and b
 
 ## Command-Line Interface Installation
 
-The Alloy command-line interface and plugin will be automatically installed by Appcelerator Studio or the Titanium CLI.
+The Alloy command-line interface and plugin will be automatically installed when you install the `alloy` package.
```

Comment:
maybe we change it to

when you install the `alloy` package by `npm i alloy`

Reply:
That is in the following `Manual Installation` section. This just says that the Alloy CLI will be installed when you do the package installation